docs/en/event-daemon/event-daemon.md

```diff
@@ -5,16 +5,25 @@ pagename: event-daemon
 lang: en
 ---
 
-
-# {% include product %}  Event Framework
-This software was originaly developed by [Patrick Boucher](http://www.patrickboucher.com) with support from [Rodeo Fx](http://rodeofx.com) and Oblique. It is now part of [{% include product %}  Software](http://www.shotgridsoftware.com)'s [open source initiative](https://github.com/shotgunsoftware).
+# Event Framework
```

Comment:
Is the missing product name here on purpose?

Reply:
Yes, sadly. Otherwise the toc doesn't work. So I stripped it.

Reply:
Looks like I removed it from the toc, so I'll add the variable back! Thanks for flagging.